python/Ganga/Core/FileWorkspace.py

```diff
@@ -13,7 +13,7 @@
 
 from Ganga.Utility.logging import getLogger
 
-import os
+from os import path, makedirs, rename, mkdir
```

Comment:
What was the reason for pulling everything out of the `os` namespace? It's so standard in Python code to do `os.path.join` or `os.rename`, when I'm seeing the code below it make me assume on first blush that `makedirs(path_to_build)` is some custom function rather than a standard library one.